BackEnd/src/main/scala/ClearBQTable.scala

```diff
@@ -0,0 +1,46 @@
+import com.google.api.services.bigquery.model.TableFieldSchema
+import com.google.cloud.dataflow.sdk.runners.DataflowPipelineRunner
```

Comment:
In this and all your files, see if you can get Eclipse to fix your imports for you: Ctrl + Shift + O